### ISSUE TITLE
Using PIC k150 to programming chip

### ISSUE THREAD

**Issue by qipsys** [tinfever/FU-Dyson-BMS#57]
Is it possible to use a PICk150 programmer instead of PICkit3 to program the chip?

**Comment by HumbleDeer** [comment]
No. The K150 programmer's firmware does not support the PIC16F18xx, the software doesn't have it, and the voltages provided by the K150 are lethal to the PIC16F18xx.
